classes/Organisations.js

```diff
@@ -21,4 +22,36 @@ module.exports = (tenant /* : Tenant */) =>
 
       return super.getRequest(`/organisations/${organisationId}`)
     }
+
+    uploadAsset(
+      assetData /* :string */,
+      assetFileName /*:string */,
+      organisationId /* : ?mixed */,
```

Comment:
We can write this in a way that does not require the organisationId passed to it. If you hit `GET /organisations` using this key and secret, only one organisation should be returned, as a key is scoped to a single organisation.